Sources/HTTP/HTTPVersion.swift

```diff
@@ -46,12 +48,11 @@ extension HTTPVersion: Comparable {
             return lhs.minor < rhs.minor
         }
     }
-
 }
 
 extension HTTPVersion: CustomStringConvertible {
     /// :nodoc:
     public var description: String {
-        return "HTTP/" + major.description + "." + minor.description
+        return "HTTP/\(major).\(minor)"
```

Comment:
Mostly did this because I think this is the style I saw generally elsewhere.  Not too sure about performance implications.